trade/trader_stakecube.js

```diff
@@ -35,13 +35,15 @@ module.exports = (
 
     module.exports.gettingMarkets = true;
     return new Promise((resolve) => {
-      stakeCubeApiClient.markets().then((markets) => {
+      stakeCubeApiClient.markets().then((scData) => {
         try {
-          if (markets.errorMessage) {
-            log.warn(`API request getMarkets(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${markets.errorMessage}`);
+          if (scData.error) {
```

Comment:
It's not necessary to check this as in case of error, function will be try-catched.
In case of error, we already logged it in `stakecube_api`.